src/pages/game/game-stats.jsx

```diff
@@ -90,10 +102,12 @@ export function GameStats({ players }) {
         <div className="message-body">
           <table className="table">
             <tbody>
-              <tr>
-                <td>Average</td>
-                <td>{hasEveryoneVoted(players) ? getAveragePoints() : '--'}</td>
-              </tr>
+              {game.votingType !== VOTING_TYPE.TSHIRT && (
```

Comment:
I don't love the `if` checks everywhere, maybe the stats for t-shirt voting type deserves it's own separate component

Reply:
yeah agree